src/Block/Block.jsx

```diff
@@ -0,0 +1,40 @@
+import './Block.css'
+import React from 'react'
+import Task from '../Task/Task'
+
+const icon1 = 'images/icon1.png';
+const icon2 = 'images/icon2.png';
+const icon3 = 'images/icon3.png';
+const icon4 = 'images/icon4.png';
+
+const Block = () => {
+    return (
+        <div className='task-block'>
+            <Task 
+                source='images/icon1.png'
```

Comment:
no need to pass the string if you save it into a variable in line 5, you can pass the variable here and it will evaluate it's value.
`source={icon1}`